templates/undercloud.conf.j2

```diff
@@ -0,0 +1,21 @@
+[DEFAULT]
+
+local_interface = {{ ctlplane_interface }}
+local_ip = 192.168.24.1/24
+{% if osp_release <= 12 %}
+network_gateway = 192.168.24.1
```

Comment:
Take the variables from group_vars as the same will be used in nic-configs. So IP addresses used in nic-configs and undercloud conf should match.